README.md

```diff
@@ -94,22 +97,60 @@ location, specify `CMAKE_INSTALL_PREFIX` when running cmake, e.g.:
-4. Download cmake and install it. Then start cmake-gui and select the STK root directory as 'Where is the source code', and a new directory in the root directory (next to src, data etc) as the build directory (for now I assume that this directory is called bld).
-5. Click on configure. You will be asked to create the directory (yes), then for your VS version. Make sure you select the right version (be aware of the easy to confuse version numbers: VS 2013 = version 12). Click on configure, then generate. This will create the directory 'bld', and a VS solution in that directory.
-6. In Visual Studio open the project file generated in the 'bld' folder.
-7. Right click on the supertuxkart project in the solution explorer, and select "Set as StartUp Project".
-8. Select Build->Build Solution (or press F7) to compile.
+## Building SuperTuxKart on Windows
+To Build SuperTuxKart on Windows, follow these instructions:
+
+1. Download and install Visual Studio from here: [Visual Studio - Download](https://www.visualstudio.com/downloads/). The free Visual Studio Community edition works fine.
+2. Download the SuperTuxKart source package from either [SuperTuxKart download area - SourceForge.net](https://sourceforge.net/projects/supertuxkart/files/SuperTuxKart/0.9.2) or [SuperTuxKart.net - Source Control](https://supertuxkart.net/Source_control), and unpack it. 
+*Note: If you downloaded the source package from here: [SuperTuxKart.net - Source Control](https://supertuxkart.net/Source_control), then both `stk-code` and `stk-assets` **must** be in the same directory, otherwise the build can result in failure*
+3. Download the Windows dependencies package from either [SuperTuxKart download area: Dependecies - SourceForge.net](https://sourceforge.net/projects/supertuxkart/files/SuperTuxKart%20Dependencies/Windows/)
+or [SuperTuxKart on GitHub - Dependencies](https://github.com/supertuxkart/dependencies), and unpack it; then, copy the `dependencies` directory from either the `windows` or the `windows_64bit` directories into the `stk-code` directory
+4. Download CMake from here: [CMake - download page](https://cmake.org/download/), install it; once CMake is installed, double click on the CMake icon on your desktop, and point it towards your `stk-code` directory in the 'Where is the source code' field, and point it to a directory called `build` or `bld`
+5. Press 'Configure'; CMake will ask you if it is OK to create the aformentioned directory, press `Yes`. CMake will then ask you about your version of Visual Studio. 
+Confirm your selection; *Please look at the table below to avoid confusion between version numbers and releases of Visual Studio*;
+CMake will begin creating the required files for the build in the directory.
+*If CMake fails to find the dependencies during configuration; you'll need to manually add them in the missing fields*
```

Comment:
I don't get that line, dependencies should be found. If they are not found then fiddling manually in cmake's variables is most likely not the right fix.